core/block_svg.js

```diff
@@ -532,6 +532,22 @@ Blockly.BlockSvg.prototype.getBoundingRectangle = function() {
   return new Blockly.utils.Rect(top, bottom, left, right);
 };
 
+
+/**
+ * Toggles the visibility of the block.
+ * @param {boolean} isVisible True if block should be visible.
+ */
```

Comment:
Currently I believe the other setVisible functions are package, because there are issues about connecting to invisible blocks and things.